projects/js-packages/components/components/zendesk-chat/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { useMemo } from 'react';
+import { getUserLocale } from '../../lib/locale';
+import utils from './utils';
+import ZendeskChatScript from './zendesk-chat-script';
+import type { ZendeskChatType } from './types';
+
+export const ZendeskChat: ZendeskChatType = () => {
+	const { isWithinAvailableChatTimes, isWithinAvailableChatDays } = utils;
+
+	const shouldShowZendeskPresalesChat = useMemo( () => {
```

Comment:
I'd love if we could fetch the available chat times from the backend in the future.

Reply:
This is a good idea, I believe an endpoint is being worked on to that provides this info, we can make that an addition later

Reply:
It looks like that endpoint might have been completed in D99766-code - I see you have already mentioned creating a follow-up task to revisit this :)